File: from-3401-to-3500/3402-minimum-operations-to-make-columns-strictly-increasing/solution.go
func minimumOperations(grid [][]int) int {
    result := 0
    for j := 0; j < len(grid[0]); j ++ {
        prev := grid[0][j]
        for i := 1; i < len(grid); i ++ {
            if grid[i][j] <= prev {
                result += prev - grid[i][j] + 1
                grid[i][j] = prev + 1
            }
            prev = grid[i][j]
        }
    }
    return result
}

File: from-3401-to-3500/3402-minimum-operations-to-make-columns-strictly-increasing/solution.py
class Solution:
    def minimumOperations(self, grid: List[List[int]]) -> int:
        result = 0
        for col in range(len(grid[0])):
            prev = grid[0][col]
            for row in range(1, len(grid)):
                if prev >= grid[row][col]:
                    result += prev - grid[row][col] + 1
                    grid[row][col] = prev + 1
                prev = grid[row][col]
        return result